pkg/log/fatalHook.go

```diff
@@ -43,8 +48,9 @@ func (f *FatalHook) Fire(entry *logrus.Entry) error {
 	}
 	filePath := filepath.Join(f.Path, fileName)
 	errDetails, _ := json.Marshal(&details)
-	// Logging information needed for error reporting -  do not modify.
-	Entry().Infof("fatal error: errorDetails%v", string(errDetails))
```

Comment:
I'm not sure but these exact logging lines are used by Monitoring team to spot errors. Might be good to check with them first